metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -119,7 +122,6 @@ object MetalsServerConfig {
       case "coc.nvim" =>
         base.copy(
           statusBar = StatusBarConfig.showMessage,
-          isHttpEnabled = true,
```

Comment:
Is this change intentional?

Reply:
Ah, no. Sorry, was playing around with this for testing. I'll revert it.